ontotext-yasgui-web-component/src/models/yasgui-configuration.ts

```diff
@@ -270,7 +277,8 @@ export const defaultYasqeConfig: Record<string, any> = {
   prefixes: {
 
   },
-  isVirtualRepository: false
+  isVirtualRepository: false,
```

Comment:
I see this is not a new code, but I wonder why its introduced here at all?!

Reply:
Done